apps/api/src/app/widgets/usecases/mark-action-as-done/update-message-actions.usecase.ts

```diff
@@ -75,9 +75,12 @@ export class UpdateMessageActions {
       );
     }
 
-    const organizationAdmin = await this.memberRepository.getOrganizationAdminAccount(command.organizationId);
-    if (organizationAdmin) {
-      this.analyticsService.track('Notification Action Clicked - [Notification Center]', organizationAdmin?._userId, {
+    const orgUser = await this.communityUserRepository.findOne({
+      _organizationId: command.organizationId,
+    });
+
+    if (orgUser) {
```

Comment:
@ChmaraX @denis-kralj-novu do we even have a _organizatonId field on the user level? I'm not sure. btw, wecan remove the orgUser._id, and just pass the org._id as theparam so we can avoid this query altogether